src/ui/hotkey_overlay.rs

```diff
@@ -285,6 +285,15 @@ fn render(
         }
     }
 
+    if config.hotkey_overlay.hide_not_bound {
+        // If hiding unbound hotkeys is desired
+        // filter out actions that aren't present in the binds
+        actions = actions
+            .into_iter()
+            .filter(|action| binds.iter().any(|bind| bind.action == **action))
+            .collect();
```

Comment:
Actually, even better: you can use https://doc.rust-lang.org/std/vec/struct.Vec.html#method.retain

Reply:
:bulb: